pkg-manager/package-requester/src/packageRequester.ts

```diff
@@ -101,7 +101,8 @@ export function createPackageRequester (
   // A lower bound of 16 is enforced to prevent performance degradation,
   // especially in CI environments. Tests with a threshold lower than 16
   // have shown consistent underperformance.
-  const networkConcurrency = opts.networkConcurrency ?? Math.max(os.cpus().length, 16)
+  // @ts-expect-error - `availableParallelism` is not exist until update @types/node to v18.14.5
```

Comment:
let's update `@types/node` then?!

Reply:
Create a new PR to update it? The version span seems a bit large, and I'm not sure if it will cause other issues.

Reply:
it is related though, so use the same PR

Reply:
Done.